core/lib/db_connection/src/lib.rs

```diff
@@ -113,7 +115,8 @@ impl Drop for PooledStorageProcessor<'_> {
             let lifetime = self.created_at.elapsed();
             CONNECTION_METRICS.lifetime[&tags.requester].observe(lifetime);
 
-            if lifetime > ConnectionPool::global_config().long_connection_threshold() {
+            // TODO: value is hardcoded for now, but should be in sync with ConnectionPool::global_config()
+            if lifetime > Duration::from_millis(5_000) {
```

Comment:
Generally I'd expect TODOs that aren't considered to be in scope for this exact PR to be labeled with a linear ticket ID.

Reply:
Also will be done in the next PR